google_analytics/preprocessing.py

```diff
@@ -76,8 +75,7 @@ def process(train, test):
     # have no way of knowning that the user will actually shop X more
     # times (or if he will visit again at all). However since this
     # info also exists in the test set we might use it.
-    total_visits = merged[["fullVisitorId", "visitNumber"]] \
-        .groupby("fullVisitorId", as_index=False).max()
+    total_visits = merged[["fullVisitorId", "visitNumber"]].groupby("fullVisitorId", as_index=False).max()
```

Comment:
Our flake8 doesn't seem to be concerned about this too much, but PEP8 specifies a max column width of 79 I believe. If we choose another length for our group, that's fine as well, but maybe good to have an explicit rule about this? I like it when I can read and/or code in two files on the same screen.

Reply:
It's not a real problem to me if we extend the limit a bit, but it is more convenient to have short lines when working on a laptop screen. 150 seems a bit much to me.

Slightly related to this: shall we agree on using a certain docstring style? I think we have some different styles in our repository now and in the long term, this could become an obstacle if we ever want to create documentation for the more general-purpose work we do.